src/S3/MultipartCopy.php

```diff
@@ -61,12 +63,11 @@ public function __construct(
         $source,
         array $config = []
     ) {
-        if (ArnParser::isArn($source)) {
-            $this->source = '';
+        if (is_array($source)) {
```

Comment:
The constructor documentation for `$source` needs to be updated to define the new behavior.

Reply:
Documentation added (pasted here so you don't have to go searching): 
```
     * @param string $source Location of the data to be copied (in the form
     *                       /<bucket>/<key>).  If the key contains a '?'
     *                       character, instead pass an array of source_key,
     *                       source_buckey, and source_version_id.
```